tests/unit/cli/admin/test_import_targets.py

```diff
@@ -0,0 +1,541 @@
+            lambda *a, **kw: fake_response
+        )
+
+        options = [
+            "-metadata-url",
+            "http://127.0.0.1/metadata/",
+            "-db-uri",
+            "postgresql://postgres:secret@127.0.0.1:5433",
+            "-csv",
+            "targets1of2.csv",
+            "-csv",
+            "targets2of2.csv",
+        ]
+        result = client.invoke(
+            import_targets.import_targets, options, obj=test_context
+        )
+        assert result.exit_code == 1, result.output
+        assert "Error 500 Internal Error" in result.output
+        assert import_targets.is_logged.calls == [
+            pretend.call("fake-server", "test-token")
+        ]
+        assert import_targets.get_headers.calls == [
+            pretend.call(test_context["settings"])
+        ]
+        import_targets.request_server.calls == [
+            pretend.call(
+                "fake-server",
+                import_targets.URL.bootstrap.value,
+                import_targets.Methods.get,
+                headers="headers",
+            )
+        ]
+
+    def test_import_targets_without_bootstrap(self, client, test_context):
+        test_context["settings"].SERVER = "fake-server"
+        test_context["settings"].TOKEN = "test-token"
+
+        import_targets.is_logged = pretend.call_recorder(
+            lambda *a: pretend.stub(state=True)
+        )
+        import_targets.get_headers = pretend.call_recorder(
+            lambda *a: "headers"
+        )
+        fake_response = pretend.stub(
+            status_code=200,
+            json=pretend.call_recorder(lambda: {"data": {"bootstrap": False}}),
+        )
+        import_targets.request_server = pretend.call_recorder(
+            lambda *a, **kw: fake_response
+        )
+
+        options = [
+            "-metadata-url",
+            "http://127.0.0.1/metadata/",
+            "-db-uri",
+            "postgresql://postgres:secret@127.0.0.1:5433",
+            "-csv",
+            "targets1of2.csv",
+            "-csv",
+            "targets2of2.csv",
+        ]
+        result = client.invoke(
+            import_targets.import_targets, options, obj=test_context
+        )
+        assert result.exit_code == 1, result.output
+        assert (
+            "import-targets` requires bootstrap process done." in result.output
+        )
+        assert import_targets.is_logged.calls == [
+            pretend.call("fake-server", "test-token")
+        ]
+        assert import_targets.get_headers.calls == [
+            pretend.call(test_context["settings"])
+        ]
+        import_targets.request_server.calls == [
+            pretend.call(
+                "fake-server",
+                import_targets.URL.bootstrap.value,
+                import_targets.Methods.get,
+                headers="headers",
+            )
+        ]
+
+    def test_import_targets_skip_publish_targets(self, client, test_context):
```

Comment:
Can we move this positive scenario closer to the rest of the positive scenarios?
What do you think?